contracts/contracts/StakingEscrow.sol

```diff
@@ -7,7 +7,7 @@ import "../aragon/interfaces/IERC900History.sol";
 import "./NuCypherToken.sol";
 import "./lib/Bits.sol";
 import "./proxy/Upgradeable.sol";
-import "../zeppelin/math/Math.sol";
+import "@openzeppelin/contracts/utils/math/Math.sol";
 import "../zeppelin/token/ERC20/SafeERC20.sol";
```

Comment:
Any reason this zeppelin contract shouldn't/can't be updated?

Reply:
I'm working on it, but it causes name collisions when i replace it with a standard import

Reply:
Fixed